pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -110,7 +110,8 @@ func NewCmdBootstrap(name, fullName string) *cobra.Command {
 	bootstrapCmd.Flags().StringVarP(&o.Prefix, "prefix", "p", "", "add a prefix to the environment names")
 	bootstrapCmd.Flags().StringVarP(&o.ImageRepo, "image-repo", "", "", "used to push built images")
 
-	bootstrapCmd.Flags().StringVarP(&o.SealedSecretsNamespace, "sealed-secrets-ns", "", "kube-system", "namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator")
+	bootstrapCmd.Flags().StringVarP(&o.SealedSecretsNamespace, "sealed-secrets-ns", "", "sealed-secrets", "namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator")
+	bootstrapCmd.Flags().StringVarP(&o.SealedSecretsController, "sealed-secrets-controller", "", "sealedsecretcontroller-sealed-secrets", "SealedSecrets Controller name (default: sealedsecretcontroller-sealed-secrets)")
```

Comment:
If you provide a default, then it's automatically output by cobra.

Running this, it's coming out with:
```
 --sealed-secrets-controller string    SealedSecrets Controller name (default: sealedsecretcontroller-sealed-secrets) (default "sealedsecretcontroller-sealed-secrets")
````

I wonder what we could do to indicate _why_ a user might want to use this, assume that the user has _not_ read the documentation.

Reply:
Removed default in help description.   Add more help messages.